salt/modules/kapacitor.py

```diff
@@ -105,6 +121,7 @@ def _run_cmd(cmd):
 def define_task(name,
                 tick_script,
                 task_type='stream',
+                dbrps=None,
```

Comment:
Can you move this new kwarg to the end of the list? We don't want to change the API unnecessarily here.